src/features/zaaksysteem/components/ZaakContactmomenten.vue

```diff
@@ -1,36 +1,13 @@
 <template>
-  <section>
+  <section class="container">
     <utrecht-heading :level="2" modelValue>Contactmomenten</utrecht-heading>
 
     <simple-spinner v-if="contactmomenten.loading" />
 
-    <template v-if="contactmomenten.success && contactmomenten.data.length">
-      <table>
-        <thead>
-          <tr>
-            <th>Datum</th>
-            <th>Medewerker</th>
-            <th>Kanaal</th>
-            <th>Gespreksresultaat</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr
-            v-for="contactmoment in contactmomenten.data"
-            :key="contactmoment.id"
-          >
-            <td>{{ formatDateOnly(new Date(contactmoment.startdatum)) }}</td>
-            <td>{{ contactmoment["x-commongateway-metadata"].owner }}</td>
-            <td>{{ contactmoment.kanaal }}</td>
-            <td>{{ contactmoment.resultaat }}</td>
-          </tr>
-        </tbody>
-      </table>
-    </template>
-
-    <span v-else-if="contactmomenten.success && !contactmomenten.data.length"
-      >Geen contactmomenten gevonden.</span
-    >
+    <contactmomenten-overzicht
+      v-if="contactmomenten.success && contactmomenten.data.length"
```

Comment:
I see. this is actually where we ignore the pagination from the gateway. I'd rather we use it here, even though it's slightly out of scope. It's not that difficult to implement and I feel we will forget it otherwise.

Reply:
Fixed